libpod/pod_api.go

```diff
@@ -12,6 +12,45 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
+// startInitContainers starts a pod's init containers.
+func (p *Pod) startInitContainers(ctx context.Context) error {
+	initCtrs, err := p.initContainers()
+	if err != nil {
+		return err
+	}
+	// Nothing to do ?
+	if initCtrLen := len(initCtrs); initCtrLen < 1 {
```

Comment:
nit: why allocate a variable here?

Reply:
thats how I roll ... no, actually previous version of the code actually used the variable but that code was removed. so this could be simplified to a simple if statement, absolutely.